Linker error on Arch

I get a Linker error on Arch:

```
libHScryptonite-0.29-717e3026048e355515ccc1fd8abeed450db0ef8d63bbf8bb8b1a7d87bbff5b11.a(ed25519.o): multiple definition of 'batch_point_buffer'

libHSed25519-donna-0.1.1-1239fe7092277c5cdd70eb45b6e85454704fc0d908ec7072245a1d50122627d5.a(ed25519.o): previous definition here

collect2: error: ld returned 1 exit status
`gcc' failed in phase `Linker'. (Exit code: 1)
```

ghc-8.6.5
cabal-3.6.2.0

--- a/README.md
+++ b/README.md
@@ -120,6 +120,17 @@ cabal v2-update
 cabal v2-build
 ```
 
+On some systems the default build might fail with linker errors complaining about cryptonite and ed25519, try:
+
+```bash
+# configure cabal 
+cabal v2-configure -f cryptonite-ed25519
+
+# Build the project.
+cabal v2-build
+```
+
+
 This will install a runnable version of Pact, which you can run via:
 
 ```bash
@@ -130,6 +141,8 @@ Alternatively, `cabal v2-install` will install the binary to `~/.cabal/bin/`, wh
 you may need to add to your path. Then, you can call `pact` as-is.
 
 #### Building with Stack
+
+***(stack is no longer supported)***
 
 Stack is a Haskell build tool that manages compiler and dependency versions for
 you. It's easy to install and use.

